lib/v2/gov/maintainer.js

```diff
@@ -100,4 +100,5 @@ module.exports = {
     '/xinyi/:path+': ['ShuiHuo'],
     '/xuzhou/hrss/:category?': ['nczitzk'],
     '/zhejiang/gwy/:category?/:column?': ['nczitzk'],
+    '/mof/bond/:category?': ['la3rence'],
```

Comment:
Please insert new entries in alphabetical order.

Reply:
fixed.